src/Settings/CustomerExperienceSettings.php

```diff
@@ -2,20 +2,35 @@
 
 namespace Astrogoat\CustomerExperience\Settings;
 
+use Astrogoat\CustomerExperience\Peripherals\Faqs;
+use Astrogoat\CustomerExperience\Peripherals\Support;
+use Astrogoat\CustomerExperience\Peripherals\SupportLinks;
 use Helix\Lego\Settings\AppSettings;
-use Illuminate\Validation\Rule;
 
 class CustomerExperienceSettings extends AppSettings
 {
-    // public string $url;
+    public bool $chat_enabled;
+    public bool $call_enabled;
+    public bool $faq_enabled;
 
-    protected array $rules = [
-        // 'url' => Rule::requiredIf($this->enabled === true),
+    public function rules()
+    {
+        return [
+          'chat_enabled' => ['nullable','boolean'],
+          'call_enabled' => ['nullable','boolean'],
+          'faq_enabled' => ['nullable','boolean'],
+        ];
+    }
+
+    protected array $peripherals = [
+        Support::class,
+        Faqs::class,
+        SupportLinks::class,
     ];
 
```

Comment:
Let's add some better labels and help for the checkboxes.

```suggestion

    public function labels(): array
    {
        return [
            'chat_enabled' => 'Enable Chat button',
            'call_enabled' => 'Enable Call button',
            'faq_enabled' => 'Enable FAQs',
        ];
    }

    public function help(): array
    {
        return [
            'chat_enabled' => 'Toggle to show the Chat button',
            'call_enabled' => 'Toggle to show the Call button',
            'faq_enabled' => 'Toggle to show the FAQ section',
        ];
    }

```